composer.json

```
@@ -14,24 +14,24 @@
         "issues": "https://github.com/acquia/http-hmac-php/issues"
     },
     "require": {
-        "php": ">=7.3.0",
+        "php": "^7.3 || ^7.4 || ^8.0 || ^8.1",
         "psr/http-message": "^1.0 || ^2.0"
     },
     "suggest": {
-        "guzzlehttp/guzzle": "^6.0",
+        "guzzlehttp/guzzle": "^6.0 || ^7.0",
         "laminas/laminas-diactoros": "^1.8 || ^2.2",
-        "symfony/security": "^4.0 || ^5.0"
+        "symfony/security": "^4.0 || ^5.0 || ^6.0"
```

Symfony 4 and 5 are several years EOL by now, and we're almost at the end of the Symfony 7 lifecycle. All the Symfony components should at minimum support Symfony 7:


```suggestion
        "symfony/security": "^4.0 || ^5.0 || ^6.0 || ^7.0"
```